libs/ledger-live-common/src/families/hedera/utils.ts

```diff
@@ -13,7 +13,11 @@ export async function getEstimatedFees(): Promise<BigNumber> {
       url: "https://countervalues.live.ledger.com/latest/direct?pairs=hbar:usd",
     });
 
-    return new BigNumber(10000).dividedBy(data[0]);
+    BigNumber.config({
+      ROUNDING_MODE: BigNumber.ROUND_UP,
+    });
+
+    return new BigNumber(10000).dividedBy(data[0]).dp(0);
```

Comment:
Can you please consider using `integerValue` instead.
I am thinking calling `BigNumber.config` could have side effects.